src/pages/Home/components/delete-movie-modal/DeleteMovieModal.module.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../../../styles/stylesVariables";
```

Comment:
You can use absolute imports `import "src/styles/stylesVariables";`

Reply:
For me more understandable use absolute imports only for modules from node_modules